### PR TITLE
Documentation cleanups

### PR DESCRIPTION
**Pull request by bkircher**
In this PR:
- tidy-up README file a bit: use proper capitalization in headings
- remove superfluous white space characters in doc sources
- fix some spelling mistakes in docs

### CODE REVIEW

File: website/docs/r/storage.html.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ The following arguments are supported:
 
     * `password_type` - (Optional) (one of plain, crypt) Required if password is set (ignored for private templates and public Windows templates).
 
-    * `sshkeys` - (Optional) (array of any - minItems: 0) Public Linux templates only! The UUIDs of SSHkeys to install for the root user.
+    * `sshkeys` - (Optional) (array of any - minItems: 0) Public Linux templates only! The UUIDs of SSH keys to install for the root user.
```
Comment (itakouna) [inline website/docs/r/storage.html.md]: The UUIDs of SSH keys to be added for the root user.

Comment (fkr) [reply website/docs/r/storage.html.md]: if we nitpick about this, imho it should be ssh keys not SSH ;)

**Review by fkr — APPROVED** [review]

File: website/docs/r/storage.html.md (hunk 1 above)
Comment (bkircher) [reply website/docs/r/storage.html.md]: @itakouna updated

Comment (bkircher) [reply website/docs/r/storage.html.md]: @fkr not sure. Lots of sites (e.g. [GitHub's help](https://help.github.com/en/github/authenticating-to-github/adding-a-new-ssh-key-to-your-github-account)) use all-uppercase for secure shell abbreviation (as it actually be the case in English language? No idea.)

Comment (fkr) [reply website/docs/r/storage.html.md]: actually @itakouna and you are correct. Even all the SSH manual pages use SSH. thanks for pointing this out!

**Pull request merged**